src/components/cards.css

```diff
@@ -200,3 +200,101 @@
   z-index: 2;
   margin: 0px 10px;
 }
+
+/* --------- small product card ------------ */
+.card-container-small {
+  width: 18.5rem;
+  box-shadow: 0px 0px 5px var(--black-200);
+  border-radius: var(--radius-md);
+  display: flex;
+  flex-direction: column;
```

Comment:
You can consider making a utility class of flex-col since you'll be using it often